index.html

```diff
@@ -81,14 +81,35 @@ <h1>
             Code, create, and <br />
             learn together
           </h1>
+
           <h2>
             Use our free, collaborative, in-browser IDE to code in 50+ languages
             — without<br />
             spending a second on setup.
           </h2>
 
           <div>
-            <button class="start-btn"><  > Start coding</button>
+            <button class="start-btn">
```

Comment:
the style of this button doesn't match the design, the arrow and "Start coding " should be inline